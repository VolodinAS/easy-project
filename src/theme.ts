const theme = {
    color: {
        typeTask: '#F8BFBF',
        typeFeature: '#B7CFFF',
        typeBug: '#FEFBBD',
        black: '#000000',
        lightPurple: '#CFC7D9',
        purple: '#7A52AA',
        lightGray: '#F5F5F5',
        darkGray: '#9C9C9C',
        red: '#EB0C0C',
        white: '#FFFFFF',
        smallText: '#999999',
        opacityGray: '#ACACAC',
        landingColors: {
            mainGray: '#D5E6EA',
            mainBlue: '#3848A9',
            benefitsItemBackground: '#EDD8BE',
            buttonStartBackground: '#E43958',
            footerBackground: '#343F53',
            headerLogo: '#010101',
            headerBackground: '#F1E7DA'
        },
        dashboardColors: {
            overlayButtonCreate: '#ABE0E7',
            overlayButtonYes: '#B4E7AB',
            overlayButtonNo: '#F9CCBE',
        }
    }
};

export default theme;
