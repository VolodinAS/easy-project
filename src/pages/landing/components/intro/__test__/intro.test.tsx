/* eslint-disable check-file/folder-naming-convention */
import React from 'react';
import { describe, it, expect, jest } from '@jest/globals';
import { render } from '@testing-library/react';


import Intro from '../intro';
import { BrowserRouter } from 'react-router-dom';
import { ThemeProvider } from '@emotion/react';
import theme from '../../../../../theme';


describe('intro', () => {
    it('video link enable', () => {
        render(
            <ThemeProvider theme={theme}>
                <BrowserRouter>
                    <Intro />
                </BrowserRouter>
            </ ThemeProvider>    
        );
        expect(document.body).toMatchSnapshot();
    });
});

