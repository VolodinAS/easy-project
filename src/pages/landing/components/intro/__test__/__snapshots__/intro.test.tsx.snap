// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`intro video link enable 1`] = `
<body>
  <div>
    <div
      class="css-5j6wn1"
      id="intro"
    >
      <div
        class="css-1v38hm"
      >
        <h1
          class="css-1s264hv"
        >
          Готовы изменить то, как вы работаете?
        </h1>
      </div>
      <div
        class="css-12c6vpr"
      >
        <p
          class="css-1l5tvrx"
        >
          Вам не нужны десятки приложений, чтобы добиться цели. Узнайте, как переход на Easy Project позволит вам управлять всеми аспектами вашего бизнеса.
        </p>
      </div>
      <div
        class="css-18gglrl"
      >
        <iframe
          class="css-k3igyc"
          src="video-link"
        />
      </div>
      <a
        class="css-11g9kr1"
        href="/easy-project/registration"
        title="Регистрация"
      >
        <button
          class="css-1haqyjf"
        >
          <div
            class="css-1o6ajog"
          >
            Начать управление проектами
          </div>
        </button>
      </a>
    </div>
  </div>
</body>
`;
